app/controllers/api/v1/list_restaurants_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::ListRestaurantsController < ApplicationController
+  def index
+    restaurants = Restaurant.all
+    render json: {entries: restaurants}
```

Comment:
Guys, you can actually view your endpoints by going to the URL (as long as it is a `get` request). See what this gives you. It is likely you want to format this response with `jbuilder`.